apps/base/utils.py

```diff
@@ -198,3 +200,31 @@ def get_queue_name(param):
         :80
     ]  # The max-length for queue-name is 80 in SQS
     return queue_name
+
+
+def send_slack_notification(webhook=settings.SLACK_WEB_HOOK_URL, message=""):
+    """
+    Send slack notification to any workspace
+    Keyword Arguments:
+        webhook {string} -- slack webhook URL (default: {settings.SLACK_WEB_HOOK_URL})
+        message {str} -- JSON/Text message to be sent to slack (default: {""})
+    """
+    try:
+        data = {
+            "text": message["text"],
+            "attachments": [
+                {
+                    "color": "ffaf4b",
+                    "fields": message["fields"]
+                }
+            ]
+        }
+        return requests.post(
```

Comment:
Why are we returning here? Is it necessary to do that? Can't we simply do `requests.post`

Reply:
@RishabhJain2018 I'm returning this so that I'm able to test in the `test_utils.py` by checking the returned object type.

Reply:
okay.